src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -109,6 +110,10 @@ def load_arguments(self, _):
         c.argument('custom_error_pages', min_api='2018-08-01', nargs='+', help='Space-separated list of custom error pages in `STATUS_CODE=URL` format.', validator=validate_custom_error_pages)
         c.argument('firewall_policy', options_list='--waf-policy', min_api='2018-12-01', help='Name or ID of a web application firewall (WAF) policy.', validator=validate_waf_policy)
 
+    with self.argument_context('network application-gateway', arg_group='Identity') as c:
+        c.argument('identity_type', help="The type of identity used for the resource.", arg_type=get_enum_type(['SystemAssigned', 'UserAssigned', 'None']))
```

Comment:
Will you support system identity in your command?

Reply:
Currently, the back-end server doesn't support system identity.

Reply:
But you provide `system identity` as a choice, it is a little confusing. I think you should add identity type in future, and we shouldn't add feature which doesn't supported now.

Reply:
I mean `identity_type` argument looks unnecessary now. We can add such argument when system identity is supported or to some degree `--identity` is enough for define identity used.

Reply:
re-think the design and I agree with you. I will move the identity_type for now.